docs/dev-guides/2-creating-an-xrpk.md

```diff
@@ -1,6 +1,6 @@
 ---
 id: creating-an-xrpk
-title: Create Your First XRPackage
+title: Create An XRPackage
 ---
 
 This guide will walk you through the basic steps for creating an XRPackage for use in the Webaverse ecosystem. The term "XRPK" stands for "XRPackage", which can be thought of as an "application" or "widget" in the Webaverse. XRPackages in the Webaverse are spatialized apps. Unlike traditional apps that exists on a 2D plane, XRPKs are applications that exist in three dimensional space.
```

Comment:
The publishing part of this document will either need to be updated to mention that the tool does not do publishing, or we need to update xrpk to actually do minting.

Reply:
c9034f4